org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/navigation/XbaseHyperLinkHelper.java

```diff
@@ -155,6 +171,30 @@ public void createHyperlinksByOffset(final XtextResource resource, final int off
 		}
 	}
 
+	/**
+	 * Creates hyperlinks on types referenced in javadoc comments.
+	 * @since 2.16
+	 */
+	protected void createHyperlinksInJavaDoc(XtextResource resource, int offset, IHyperlinkAcceptor acceptor) {
+		ILeafNode node = NodeModelUtils.findLeafNodeAtOffset(resource.getParseResult().getRootNode(), offset);
+		EObject semanticObject = NodeModelUtils.findActualSemanticObjectFor(node);
+		List<ReplaceRegion> replaceRegions = javaDocTypeReferenceProvider.computeTypeRefRegions(node);
+		for(ReplaceRegion replaceRegion : replaceRegions) {
+			if(replaceRegion.getOffset()<=offset && offset<=replaceRegion.getEndOffset()) {
```

Comment:
Small style remark, spaces around operators. If anything, it's consistent with the rest of your code